spec/validators/income_assessment/answers_validator_spec.rb

```diff
@@ -207,32 +226,41 @@
 
     context 'when has_no_income_payments is no and income_payments are present but not complete' do
       before do
-        allow(record).to receive(:has_no_income_payments).and_return('no')
+        record.has_no_income_payments = 'no'
```

Comment:
I'm not sure it's possible for this to get set to 'no' is it? - I assume it would return nil, unless types.include?('none')